client/jetpack-cloud/sections/manage/pricing/controller.tsx

```diff
@@ -1,8 +1,22 @@
 import { type Callback } from '@automattic/calypso-router';
+import { getQueryArgs, addQueryArgs } from '@wordpress/url';
 import ManagePricingPage from 'calypso/jetpack-cloud/sections/manage/pricing/primary';
 
+export const JETPACK_COM_A4A_LANDING_PAGE = 'https://jetpack.com/for-agencies/';
```

Comment:
I noted this in the other PR but I think we can move this to a new constants file 😄

Reply:
Yes, Thanks Dylan! 🙂 I moved it to a constants file in the other PR, so then I will rebase this PR to that one. 👍